### PR TITLE
Include Azure as a new MC target in configuration

### PR DESCRIPTION
**Pull request by CarlosCortizasCT**
#### Summary

Include Azure as a new MC target in configuration.

#### Description

We add a new configuration option for Custom Applications and Custom Views so they can set the upcoming Azure production environment as a target.

### CODE REVIEW

File: packages/application-config/src/constants.ts [hunk 1]
```diff
@@ -14,6 +15,7 @@ export const MC_API_URLS = {
   AWS_FRA: 'https://mc-api.eu-central-1.aws.commercetools.com',
   AWS_OHIO: 'https://mc-api.us-east-2.aws.commercetools.com',
   AWS_CN: 'https://mc-api.cn-northwest-1.aws.commercetools.cn',
+  AZURE_US: 'https://mc-api.eastus1.azure.commercetools.com',
```
Comment (kark) [inline packages/application-config/src/constants.ts:18]:
```suggestion
  AZURE_US: 'https://mc-api.eastus.azure.commercetools.com',
```

File: packages-backend/express/src/constants.ts [hunk 2]
```diff
@@ -14,6 +15,7 @@ export const MC_API_URLS = {
   AWS_FRA: 'https://mc-api.eu-central-1.aws.commercetools.com',
   AWS_OHIO: 'https://mc-api.us-east-2.aws.commercetools.com',
   AWS_CN: 'https://mc-api.cn-northwest-1.aws.commercetools.cn',
+  AZURE_US: 'https://mc-api.eastus1.azure.commercetools.com',
```
Comment (emmenko) [inline packages-backend/express/src/constants.ts:18]:
```suggestion
  AZURE_US: 'https://mc-api.eastus.azure.commercetools.com',
```

File: packages/application-config/src/constants.ts (hunk 1 above)
Comment (CarlosCortizasCT) [reply packages/application-config/src/constants.ts]: Updated here: 39f48fbd7fd45ea6dcf9563638f4af2830881bb1

File: packages-backend/express/src/constants.ts (hunk 2 above)
Comment (CarlosCortizasCT) [reply packages-backend/express/src/constants.ts]: Updated here: 39f48fbd7fd45ea6dcf9563638f4af2830881bb1

**Review by emmenko — COMMENTED** [review]
We also need to map the values in the code:

* for the application config: https://github.com/commercetools/merchant-center-application-kit/blob/8c10dd837ffb27b20b40c1596707c64a96c7d41b/packages/application-config/src/utils.ts#L5-L30
* in the express middleware: https://github.com/commercetools/merchant-center-application-kit/blob/8c10dd837ffb27b20b40c1596707c64a96c7d41b/packages-backend/express/src/auth.ts#L54-L73

**Comment by CarlosCortizasCT** [comment]
> We also need to map the values in the code:
> 
> * for the application config: https://github.com/commercetools/merchant-center-application-kit/blob/8c10dd837ffb27b20b40c1596707c64a96c7d41b/packages/application-config/src/utils.ts#L5-L30
> * in the express middleware: https://github.com/commercetools/merchant-center-application-kit/blob/8c10dd837ffb27b20b40c1596707c64a96c7d41b/packages-backend/express/src/auth.ts#L54-L73

Updated here: 397fa78429285f03192541bfef31d4cb6a770b6d

**Review by kark — APPROVED** [review]
Thanks 🙌

**Review by Rhotimee — APPROVED** [review]
🙌🏽

**Review by emmenko — APPROVED** [review]
Perfect, thanks!

**Pull request merged**